applications/GeoMechanicsApplication/python_scripts/geomechanics_solver.py

```diff
@@ -361,6 +361,11 @@ def _add_temperature_variables(self):
         self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.DT_TEMPERATURE)
         # Add variables for the heat conditions
         self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.NORMAL_HEAT_FLUX)
+        self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.AIR_TEMPERATURE)
```

Comment:
Let's make an issue to make these additions conditional (e.g. only when we use the t_solver?)